websocket_clients.py

```diff
@@ -31,17 +41,115 @@
 # }
 _wsGroups = {}
 
-def AddClient(userId, ws):
+_userIdSelf = '_landapp'
```

Comment:
I don't think we ended up using any of this :( so we can remove all these changes.